python/README.md

```diff
@@ -4,3 +4,30 @@ indy-agent implementation in Python
 This is an implementation of indy-agent written in Python.
 
 This agent seeks to be as simple as possible while accurately representing the Sovrin protocol.
+
+Requirements
+------------
+- Python 3.6
+- Latest `libindy` from https://github.com/hyperledger/indy-sdk
+
+Quickstart
+----------
+
+First, create a virtual environment to install indy-agent dependencies:
+
+```sh
+cd python/
```

Comment:
This needs a little more depth. which path are we expected to be in here? Obviously we won't know the full path, but we can reference something like where the code was checked out, or some other path the user has created for a specific purpose.